wkconnect/backends/neuroglancer/sharding.py

```diff
@@ -103,6 +101,25 @@ def __post_init__(self) -> None:
             self, "_minishard_mask", compute_minishard_mask(self.minishard_bits)
         )
 
+    @staticmethod
+    def parse(
+        info_json: Any,
+        dataset_size: Optional[Vec3D] = None,
+        chunk_size: Optional[Vec3D] = None,
+    ) -> "ShardingInfo":
+        assert info_json["@type"] == "neuroglancer_uint64_sharded_v1"
```

Comment:
What happens if the assertions are not green? Is this propagated to the user? Should there be readable assertion messages?